bot/cogs/grades_data/grades_cog.py

```diff
@@ -318,7 +319,7 @@ async def grades(self, ctx, course, year=MIN_YEAR):
                 embed = discord.Embed(title="Grades", color=Colors.ClemsonOrange)
                 result = self.go(course, year)
                 embed.add_field(name="Result", value=result, inline=False)
-                prefix = (await self.bot.get_prefix(ctx))[2]
+                prefix = await self.bot.get_prefix(ctx)
```

Comment:
i put this here this morning to account for some changes i made to how clembot does prefixes 
```suggestion
                prefix = (await self.bot.get_prefix(ctx))[2]
```